tests/regression/phpmyadmin-plugin/9513101.yaml

```diff
@@ -0,0 +1,23 @@
+---
+meta:
+  author: "azurit"
+  description: "Test PhpMyAdmin Plugin"
```

Comment:
This should be ``PhpMyAdmin Rule Exclusions Plugin`` by default unless there's a reason to change it
```suggestion
  description: "PhpMyAdmin Rule Exclusions Plugin"
```